Cargo.toml

```diff
@@ -46,6 +46,7 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 structopt = "0.3.17"
 uint = "0.8.3"
+web3 = { git = "https://github.com/tomusdrw/rust-web3" }
```

Comment:
Crate can not publish to `crates.io` if using `git` resource.

Reply:
I'll change to direct http request